pytest_factoryboy/fixture.py

```diff
@@ -251,12 +250,14 @@ def model_fixture(request: SubRequest, factory_name: str) -> Any:
     # Try to evaluate as much post-generation dependencies as possible
     factoryboy_request.evaluate(request)
 
-    fixture_name = str(request.fixturename)
-    factory_class: FactoryType = request.getfixturevalue(factory_name)
+    assert request.fixturename  # NOTE: satisfy mypy
+    fixture_name = request.fixturename
     prefix = "".join((fixture_name, SEPARATOR))
+    # NOTE: following type hinting is required, because of `mypy` bug.
+    # Reference: https://github.com/python/mypy/issues/2477
+    factory_class: factory.FactoryMetaClass = request.getfixturevalue(factory_name)
```

Comment:
factory.FactoryMetaClass does not exist, did you mean factory.base.FactoryMetaClass?

This shows how this annotation is completely ignored by mypy, yet this is an accepted workaround 🤷‍♂️